src/main/java/com/api/models/UserFactory.java

```diff
@@ -15,4 +15,13 @@ public static Map<String, String> createDefaultUserData() {
         data.put("status", TestDataConfig.get("user.status"));
         return data;
     }
+
+    public static Map<String, String> createInvalidUserData() {
```

Comment:
The values inside createInvalidUserData() are currently hardcoded. It reduces flexibility and reusability. Consider externalizing the invalid test data (enums) to make the method more maintainable and easier to scale for future test cases with varying types of invalid input.